src/hooks/useFeePanelInputRefs.ts

```diff
@@ -1,6 +1,6 @@
 import { useCallback, useRef } from 'react';
 import { TextInput } from 'react-native';
-import useMagicAutofocus from './useMagicAutofocus';
+import useMagicAutofocus from '@/hooks/useMagicAutofocus';
```

Comment:
this file was renamed `useFeePanelInputRefs` and is missing the `s` after `Fee` in `useFeesPanelInputRefs` which is what is still being used everywhere else